### ISSUE TITLE
PPTs failing accessibility test: Missing Slide Title

### ISSUE THREAD

**Issue by bob-dlc** [gitbrent/PptxGenJS#1298]
The accessibility checker built into PowerPoint reports that slides are "Missing Slide Title" even when I add a slide title as described in Issue #903.
